.gitignore

```diff
@@ -1,4 +1,4 @@
-bower_components*
-bower-*
 node_modules
 package-lock.json
+
+node_modules
```

Comment:
I noticed the converter adds node_modules here even if it already exists, so it's duplicated. Probably doesn't hurt anything though.